app/controllers/application_controller.rb

```diff
@@ -1,4 +1,16 @@
 class ApplicationController < ActionController::Base
   protect_from_forgery
   include SessionsHelper
+
+  def hello_world
+    if this_is_a_really_really_really_really_really_really_really_really_really_long_variable
+      true
+    else
+      'false'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.